watch/app/src/main/java/com/google/samples/quickstart/engagesdksamples/watch/publish/ClusterRequestFactory.kt

```diff
@@ -40,6 +41,7 @@ class ClusterRequestFactory(context: Context) {
 
   private val db = WatchDatabase.getDatabase(context, CoroutineScope(SupervisorJob()))
   private val movieDao = db.movieDao()
+  private val accountProfile = AccountProfile("account_id", "profile_id")
```

Comment:
Let's use `AccountProfile.Builder` instead.

Reply:
Changed to use Builder